src/main/java/frc/robot/subsystems/Rotors.java

```diff
@@ -13,28 +13,28 @@
 public class Rotors extends SubsystemBase {
   /** Creates a new Rotors. */
   final TalonFX intakeMotor = new TalonFX(mapRotors.INTAKE_CAN);
-  final TalonFX funnelBeltMotor = new TalonFX(mapRotors.FUNNEL_BELT_CAN);
-  final TalonFX rightflywheelMotor = new TalonFX(mapRotors.RIGHT_FLYWHEEL_CAN);
-  final TalonFX leftflywheelMotor = new TalonFX(mapRotors.LEFT_FLYWHEEL_CAN);
+  final TalonFX hopperRightMotor = new TalonFX(mapRotors.HOPPER_RIGHT_CAN);
+  final TalonFX hopperLeftMotor = new TalonFX(mapRotors.HOPPER_LEFT_CAN);
+  final TalonFX flywheelMotor = new TalonFX(mapRotors.FLYWHEEL_CAN);
 
   public Rotors() {
     intakeMotor.getConfigurator().apply(ConstRotors.INTAKE_CONFIGURATION);
-    funnelBeltMotor.getConfigurator().apply(ConstRotors.FUNNEL_BELT_CONFIGURATION);
-    rightflywheelMotor.getConfigurator().apply(ConstRotors.FLYWHEEL_CONFIGURATION);
-    leftflywheelMotor.getConfigurator().apply(ConstRotors.FLYWHEEL_CONFIGURATION);
+    hopperRightMotor.getConfigurator().apply(ConstRotors.HOPPER_RIGHT_CONFIGURATION);
+    hopperLeftMotor.getConfigurator().apply(ConstRotors.HOPPER_LEFT_CONFIGURATION);
+    flywheelMotor.getConfigurator().apply(ConstRotors.FLYWHEEL_CONFIGURATION);
   }
 
   public void setIntakeMotorSpeed(double speed) {
     intakeMotor.set(speed);
   }
 
   public void setFunnelBeltMotorSpeed(double speed) {
```

Comment:
should this still be named FunnelBeltMotor?

Reply:
also looks like you are just setting right motor here? Should we set both in here or make separate functions for left and right each?